src/components/DestinationCard/DestinationCard.scss

```diff
@@ -1,29 +1,35 @@
 @import "~sass-rem";
 
 .destination {
-  height: rem(238px);
-  padding: rem(0) rem(20px);
-  box-sizing: border-box;
-  cursor: pointer;
+  margin-bottom: 20px;
```

Comment:
same here convert px to rem using rem(20px)

Reply:
Done.